neon_utils/packaging_utils.py

```diff
@@ -265,3 +266,26 @@ def get_skill_license():  # TODO: Implement OSM version of this
     skill_data = dict_merge(default_skill, skill_data)
     skill_data["requirements"]["python"].sort()
     return dict(dict_merge(skill_data, readme_data))
+
+def install_packages_from_pip(module: str, packages: List[str]) -> int:
```

Comment:
Please add some kind of test coverage in [packaging_util_tests.py](https://github.com/NeonGeckoCom/neon-utils/blob/dev/tests/packaging_util_tests.py#L37). Consider mocking `pip.main` and checking for a valid call